File: app/src/main/java/bg/o/sim/colourizmus/view/DashboardActivity.java
package bg.o.sim.colourizmus.view;

import android.app.Activity;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;
import android.widget.ImageButton;

import bg.o.sim.colourizmus.R;

public class DashboardActivity extends Activity{

    private ImageButton mColourCreationButton, mGalleryButton;

    private final View.OnClickListener mClickListener = new View.OnClickListener() {
        @Override
        public void onClick(View v) {
            switch (v.getId()){
                case R.id.dashboard_button_colour_creation:
                    ColourCreationActivity.start(DashboardActivity.this);
                    break;
                case R.id.dashboard_button_gallery:
                    ColourListActivity.start(DashboardActivity.this);
                    break;
            }
        }
    };

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_dashboard);

        mColourCreationButton = findViewById(R.id.dashboard_button_colour_creation);
        mGalleryButton = findViewById(R.id.dashboard_button_gallery);

        mColourCreationButton.setOnClickListener(mClickListener);
        mGalleryButton.setOnClickListener(mClickListener);
    }
}

File: app/src/main/java/bg/o/sim/colourizmus/view/DashboardActivity.kt
package bg.o.sim.colourizmus.view

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import bg.o.sim.colourizmus.R
import bg.o.sim.colourizmus.databinding.ActivityDashboardBinding

class DashboardActivity : Activity() {

    private lateinit var binding: ActivityDashboardBinding

    private val mClickListener = View.OnClickListener { v ->
        when (v.id) {
            R.id.dashboard_button_colour_creation -> startActivity(Intent(this, ColourCreationActivity::class.java))
            R.id.dashboard_button_gallery -> startActivity(Intent(this, ColourListActivity::class.java))
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityDashboardBinding.inflate(layoutInflater);
        setContentView(binding.root)

        binding.dashboardButtonColourCreation.setOnClickListener(mClickListener)
        binding.dashboardButtonGallery.setOnClickListener(mClickListener)

//        dashboard_button_gallery.background = CharDrawable('c', Color.rgb(180, 10, 10), Color.rgb(10, 10, 180))
//        dashboard_button_colour_creation.background = CharDrawable('T', Color.rgb(10, 180, 10), Color.rgb(180, 10, 180))
    }
}